installation/kubernetes.md

```diff
@@ -144,10 +144,49 @@ spec:
 
 ### Configure Fluent Bit
 
-Assuming the basic volume configuration described previously, you can apply the
-following configuration to start logging:
+Assuming the basic volume configuration described previously, you can apply one of the following configurations to start logging:
+
+{% tabs %}
+{% tab title="fluent-bit.yaml" %}
 
 ```yaml
+parsers:
+    - name: docker
+      format: json
+      time_key: time
+      time_format: '%Y-%m-%dT%H:%M:%S.%L'
+      time_keep: true
+      
+pipeline:
+    inputs:
+        - name: tail
+          tag: kube.*
+          path: 'C:\\var\\log\\containers\\*.log'
+          parser: docker
+          db: 'C:\\fluent-bit\\tail_docker.db'
+          mem_buf_limit: 7MB
+          refresh_interval: 10
+          
+        - name: tail
+          tag: kube.error
+          path: 'C:\\k\\kubelet.err.log'
+          db: 'C:\\fluent-bit\\tail_kubelet.db'
```

Comment:
I think we need a volume for this today too otherwise it's ephemeral

Reply:
I can't find a volume key in the tail plugin docs... what do you mean?